spec/requests/user/user_profile_spec.rb

```diff
@@ -159,4 +159,20 @@
       expect(response.body).to redirect_to "/internal/users/#{user.id}"
     end
   end
+
+  describe "redirect to lowercase route" do
+    let(:user) { create(:user) }
+
+    before do
+      get "/#{user.username.capitalize}"
+    end
+
+    it "redirects to the lowercase route for usernames" do
+      expect(response.body).to include("/#{user.username.downcase}")
```

Comment:
What you can do instead is 
```
expect(response.body).to redirect_to some_correct_path
```

but of course, your test should be in `spec/requests/stories_index_spec.rb` instead